packages/clerk-js/src/ui/components/PricingTable/PricingTableDefault.tsx

```diff
@@ -64,3 +83,417 @@ export function PricingTableDefault({
     </InternalThemeProvider>
   );
 }
+
+/* -------------------------------------------------------------------------------------------------
+ * Card
+ * -----------------------------------------------------------------------------------------------*/
+
+interface CardProps {
+  plan: __experimental_CommercePlanResource;
+  planPeriod: __experimental_CommerceSubscriptionPlanPeriod;
+  setPlanPeriod: (p: __experimental_CommerceSubscriptionPlanPeriod) => void;
+  onSelect: (plan: __experimental_CommercePlanResource) => void;
+  isCompact?: boolean;
+  props: __experimental_PricingTableProps;
+}
+
+function Card(props: CardProps) {
+  const { plan, planPeriod, setPlanPeriod, onSelect, props: pricingTableProps, isCompact = false } = props;
+  const isDefaultLayout = pricingTableProps.layout === 'default';
+  const ctaPosition = (isDefaultLayout && pricingTableProps.ctaPosition) || 'top';
+  const collapseFeatures = (isDefaultLayout && pricingTableProps.collapseFeatures) || false;
+  const { id, slug, subscriptionIdForCurrentSubscriber, features } = plan;
+  const totalFeatures = features.length;
+  const hasFeatures = totalFeatures > 0;
+
+  return (
+    <Box
+      key={id}
+      elementDescriptor={descriptors.pricingTableCard}
+      elementId={descriptors.pricingTableCard.setId(slug)}
+      sx={t => ({
+        display: 'flex',
+        flexDirection: 'column',
+        background: common.mergedColorsBackground(
+          colors.setAlpha(t.colors.$colorBackground, 1),
+          t.colors.$neutralAlpha50,
+        ),
+        borderWidth: t.borderWidths.$normal,
+        borderStyle: t.borderStyles.$solid,
+        borderColor: t.colors.$neutralAlpha100,
+        boxShadow: !isCompact ? t.shadows.$cardBoxShadow : undefined,
+        borderRadius: t.radii.$xl,
+        overflow: 'hidden',
+        textAlign: 'left',
+      })}
+      data-variant={isCompact ? 'compact' : 'default'}
+    >
+      <CardHeader
+        plan={plan}
+        isCompact={isCompact}
+        isActivePlan={!!subscriptionIdForCurrentSubscriber}
+        planPeriod={planPeriod}
+        setPlanPeriod={setPlanPeriod}
+      />
+      <ReversibleContainer reverse={ctaPosition === 'top'}>
+        {!collapseFeatures && hasFeatures ? (
+          <Box
+            elementDescriptor={descriptors.pricingTableCardFeatures}
+            sx={t => ({
+              display: 'flex',
+              flexDirection: 'column',
+              flex: '1',
+              padding: isCompact ? t.space.$3 : t.space.$4,
+              backgroundColor: t.colors.$colorBackground,
+              borderTopWidth: t.borderWidths.$normal,
+              borderTopStyle: t.borderStyles.$solid,
+              borderTopColor: t.colors.$neutralAlpha100,
+            })}
+            data-variant={isCompact ? 'compact' : 'default'}
+          >
+            <CardFeaturesList
+              plan={plan}
+              isCompact={isCompact}
+            />
+          </Box>
+        ) : null}
```

Comment:
Is this a pattern here to always use a ternary returning null rather than an `&&` condition? 🤔

Reply:
Out of habit based on this post https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
Definitely a safer choice 👍🏼